### ISSUE TITLE
[all] Fix new warnings detected by Flutter Analyze

### ISSUE THREAD

**Issue by miquelbeltran** [fluttercommunity/plus_plugins#119]
- 'RaisedButton' is deprecated and shouldn't be used. Use ElevatedButton instead.
- A value of type 'void' can't be returned by the 'onError' handler because it must be assignable to 'FutureOr<...>'

**Comment by ollydixon** [comment]
How simpletons like me who don't use the material pallet (There are many others)

How do we now set these colors with our simple self created style guides?

Here's an example;

Our app colors are defined like this (Because material had some bugs)

```
class AppColors {
  static const Color darkFont = Color(0xff242424);
  static const Color greyFont = Color(0xFF7f7e7f);
  static const Color fontDescriptionDark = Color(0xff4a494a);
  static const Color lightFont = Color(0xffffffff);
  static const Color main = Color(0xFFe14736);
  static const Color mainFaded = Color(0xFFd98389);
  static const Color altColor = Color(0xFF322a2c);
  static const Color backgroundGreyDark = Color(0xFF171819);
  static const Color greyLine = Color(0xFF373638);
  static const Color grey = Color(0xFFF5F5F5);
```

Now 'ElevatedButton' (which is a really werid name, raised was much better from a logical [English sense](https://diffsense.com/diff/elevate/raise#:~:text=When%20used%20as%20verbs%2C%20elevate,of%20materials%20or%20constituent%20parts.), I guess this was some virtual signalling crap from Google? ) requires some 'MaterialStateProperty'

Basically, I can figure out how to replace this using a simple StyleGuide.

```
ElevatedButton(
          onPressed: () {
            if (_formKey.currentState.validate()) {
              widget.onName(_workoutNameController.text);
            }
          },
          style: ButtonStyle(
            backgroundColor: Colors.red,
          ),
          color: Colors.red,
          child: Text(
            lang('Save'),
            style: TextStyle(
              color: Colors.white,
              fontSize: 18,
              fontWeight: FontWeight.w700,
            ),
          ),
        )
```
